docs/customization.md

```diff
@@ -0,0 +1,7 @@
+# Customization
+
+The AsyncAPI Model SDK uses for customization **presets**.
+
+## Preset
+
+Each **preset** is a JS object which extends the rendered model with additional content. 
```

Comment:
Why is this added in this pr? 😄

Reply:
Mistake 😄 I forgot to checkout branch and by mistake pushed new docs.